app/src/test/java/io/hstream/testing/BasicTest.java

```diff
@@ -97,13 +101,13 @@ public void teardown() throws Exception {
     hStreamClient.close();
   }
 
-  void produce(Producer p, int tid) throws InterruptedException {
+  void produce(Producer p, int tid) throws Exception {
     Random rand = new Random();
     byte[] rRec = new byte[128];
     for (int i = 0; i < 1000; i++) {
       logger.info("Thread " + tid + " write");
       rand.nextBytes(rRec);
-      p.write(rRec).join();
+      p.write(Record.newBuilder().rawRecord(rRec).build()).join();
```

Comment:
Use the helper function `buildRecord()` in `TestUtil` to avoid writing this directly.